packages/kolibri/components/CoreTable.vue

```diff
@@ -152,8 +160,12 @@
   }
 
   /deep/ td {
+    display: inline-block;
     max-width: 300px;
+    padding-bottom: 4px;
     overflow-x: auto;
+    text-overflow: ellipsis;
```

Comment:
I think this would undo something that I suspect was done intentionally to wrap text within the `td` element rather than hide it by an ellipsis.